docs/modules/ROOT/pages/echo.adoc

```diff
@@ -0,0 +1,19 @@
+= echo
+
+Echo command prints the given string.
+
+Echo command can only be used in the interactive and scripting modes.
```

Comment:
for consistency with def.adoc:

NOTE: This command can only be used in interactive and batch mode.

(I believe batch is the right  @yuce can confirm

Reply:
I am pretty sure batch is not correct. We have the same note for `alias` command as well.

Reply:
https://github.com/hazelcast/clc-docs/pull/5/commits/156434bc9a0369bc6690f0c7071ef6942ee01478

Reply:
We might need to adjust the docs a bit - https://docs.hazelcast.com/clc/5.3.5/overview#modes gives only two modes: interactive and non-interactive. Scripting is mentioned as used in non-interactive mode (https://docs.hazelcast.com/clc/5.3.5/scripting-for-automation).

We have to be consistent in terminology to avoid confusion - not for us to decide right now, however. I can sort it out as a separate exercise.

Reply:
Shall we make this change in another PR? If there isn't anything else shall we merge this PR?

Reply:
Yes, this will be another PR, as we'd have to do a more thorough review and decide on the consistent terminology.

I'll approve now and this can be merged.